hw2/05-form.html

```diff
@@ -7,14 +7,139 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <!-- To be able to use js for modal-->
+    <script
+      defer
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    ></script>
     <script defer src="05-form.js"></script>
   </head>
-  <body>
+  <body class="bg-dark">
     <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
-      </form>
+      <div class="container">
+        <div class="row mt-4 justify-content-center">
+          <div class="col-8 p-3 fw-bold bg-light">
+            <form id="form">
+              <h1 class="text-center mb-5">Exercise 05 - Form</h1>
+              <div class="mb-3">
+                <label for="name" class="form-label">Full Name:</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="name"
+                  name="Full Name"
+                />
+              </div>
+              <div class="mb-3">
+                <label for="email" class="form-label">Email:</label>
+                <input
+                  type="email"
+                  class="form-control"
+                  id="email"
+                  name="Email"
+                />
+              </div>
+              <div class="mb-3">
+                <label for="select">Registration Status:</label>
+                <select
+                  class="form-select"
+                  id="select"
+                  name="Registration Status"
+                >
+                  <option selected>Choose an Option</option>
+                  <option>Open</option>
+                  <option>Closed</option>
+                </select>
+              </div>
+              <div class="mb-3">
+                <p>Have you taken any of the following courses?</p>
+                <div class="form-check fw-normal">
```

Comment:
Consider wrapping your checkboxes in a fieldset to add structure to the html

Reply:
I have added a fieldset to wrap the checkboxes. Since it needs a legend because there is an accessibility issue without one, I was able to replace the p element so it worked out.